src/App.jsx

```diff
@@ -110,7 +110,7 @@ function App() {
                   <ProtectedRouteWithProps exact path='/auth' component={AuthCallback} setLogin={setLogin} />
                   <ProtectedRouteWithProps exact path="/" component={Home} />
                   <ProtectedRouteWithProps path="/sites/:siteName/collections/:collectionName/:fileName" component={EditPage} isCollectionPage={true} isResourcePage={false} />
-                  <ProtectedRouteWithProps path="/sites/:siteName/collections/:collectionName" component={CollectionPages} />
+                  <ProtectedRouteWithProps path="/sites/:siteName/collections/:collectionName" component={CollectionPages} isResource={false}/>
```

Comment:
Can we rename `CollectionPages` to `CategoryPages` in this file? I think I did it in the other PR but it's not showing up

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/207/commits/31dd245159815cafda93bfa43ccba598cea34c96